src/oscar/apps/basket/abstract_models.py

```diff
@@ -66,6 +66,7 @@ class AbstractBasket(models.Model):
     class Meta:
         abstract = True
         app_label = 'basket'
+        ordering = ['-date_created']
```

Comment:
Thinking about this some more and I can't find a reasonable use case for wanting to query and display baskets in any particular order. I think we can probably do without ordering here.